spec/data/boolean.yaml

```diff
@@ -1,5 +1,6 @@
 form:
   title: "Bool test"
+  id: 7e6f5289d887d8afb01b17e044e9c148f4b8d4e5
   version: "1"
```

Comment:
really? That seemed like overkill to me. When is that info ever going to be useful?

I very nearly killed the version attribute in its entirety; I think the id field in the database is better in every way.

What's the use case?